src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -338,13 +325,31 @@ public void GetIssues_NotConnected_NoErrors()
             // Arrange
             SetupSolutionBinding(isConnected: false, issues: null);
 
-            // Act
+            int callCount = 0;
```

Comment:
Test for the new behaviour: start the initial fetch, then dispose, then check that getting issues returns an empty list.
This race condition could happen in practice, but only when a solution is being closed so it won't affect the analyzer results as they will be discarded at that point.  m,